### PR TITLE
style: Added PT translation

### LINKED ISSUE #13: Add Portuguese Translations
**Is your feature request related to a problem? Please describe.**
This library is going to be used mostly by people from Mozambique and with that assumption in mind, I think we should add Portuguese translations to the strings that will be picked up automatically if the user has Portuguese as the priority language.

**Describe the solution you'd like**
Add string resource variant for Portuguese.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by EdNgulele**
I used the Android Studio Translation Editor to add the translation for all string resources inside the UI module.
This should fix #13

### CODE REVIEW

File: ui/src/main/res/values-pt-rPT/strings.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <string name="payment_activity_toolbar_title">Faça o Pagamento</string>
+    <string name="payment_card_pay_button_text">Pague com o M-Pesa</string>
+    <string name="payment_card_view_phone_number_prompt">Insira o número de telefone da sua conta M-Pesa</string>
+    <string name="payment_authentication_card_title">Verifique o seu telefone</string>
+    <string name="payment_authentication_card_description">Uma notificação USSD foi enviada para o número: %s \n\nInsira o PIN do seu M-Pesa para completar a transação.</string>
+    <string name="payment_authentication_card_learn_more_button_text">Saiba mais</string>
+    <string name="payment_authentication_card_timer_seconds_text">Segundos</string>
+    <string name="payment_error_card_title">O Pagamento Falhou</string>
+    <string name="payment_error_card_retry_button">Tente Novamente</string>
+    <string name="payment_response_ins_0">Pagemento Concluído com Sucesso.</string>
```
Comment (rosariopfernandes) [inline ui/src/main/res/values-pt-rPT/strings.xml]: typo: ~Pagemento~ Pagamento

File: ui/src/main/res/values-pt-rPT/strings.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <string name="payment_response_ins_unhandled_response">Algo deu errado ao processar o seu pagamento.</string>
+    <string name="payment_response_ins_2006">O pagamento falhou devido ao saldo insuficiente na sua conta.</string>
+    <string name="payment_response_ins_5">Pagemento cancelado. Tente novamente se quiser alterar o número da conta M-Pesa.</string>
+    <string name="payment_response_ins_4">Pagamento falhou. O número de telefone inserido não tem uma conta M-Pesa válida.</string>
+    <string name="payment_response_ins_9">"A solicitação do pagamento atingiu o tempo limite porque o PIN de autorização não foi insirido a tempo. "</string>
```
Comment (rosariopfernandes) [inline ui/src/main/res/values-pt-rPT/strings.xml]: ~insirido~ inserido

Comment (rosariopfernandes) [reply ui/src/main/res/values-pt-rPT/strings.xml]: In this same line: Why do we have it between quotation marks? I don't see it on the english version.

**Comment by EdNgulele** [comment]
Thank you very much @rosariopfernandes for the review. I'll fix them shortly.

File: ui/src/main/res/values-pt-rPT/strings.xml (hunk 1 above)
Comment (rosariopfernandes) [inline ui/src/main/res/values-pt-rPT/strings.xml]: ~Pagemento~ Pagamento

Comment (rosariopfernandes) [inline ui/src/main/res/values-pt-rPT/strings.xml]: ~Pagemento~ Pagamento

**Pull request merged**